<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OVSDB Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Bridge">
         <attribute name="title">
          <string>Bridge</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTableWidget" name="BridgeTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Controller">
         <attribute name="title">
          <string>Controller</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableWidget" name="ControllerTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Interface">
         <attribute name="title">
          <string>Interface</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="InterfaceTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Mirror">
         <attribute name="title">
          <string>Mirror</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableWidget" name="MirrorTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="NetFlow">
         <attribute name="title">
          <string>NetFlow</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTableWidget" name="NetFlowTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Open_vSwitch">
         <attribute name="title">
          <string>Open_vSwitch</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QTableWidget" name="Open_vSwitchTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Port">
         <attribute name="title">
          <string>Port</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QTableWidget" name="PortTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="QoS">
         <attribute name="title">
          <string>QoS</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QTableWidget" name="QoSTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Queue">
         <attribute name="title">
          <string>Queue</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QTableWidget" name="QueueTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sFlow">
         <attribute name="title">
          <string>sFlow</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QTableWidget" name="sFlowTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="SSL">
         <attribute name="title">
          <string>SSL</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QTableWidget" name="SSLTable"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="hostLabel">
          <property name="text">
           <string>Host</string>
          </property>
          <property name="buddy">
           <cstring>hostComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="hostComboBox">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="intervalCheckBox">
          <property name="text">
           <string>Auto-refetch every</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="intervalSpinBox">
          <property name="suffix">
           <string>s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="fetchButton">
          <property name="text">
           <string>Refetch</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_DB_Window"/>
    <addaction name="actionNew_Flow_Window"/>
    <addaction name="actionShow_Log"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Log">
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="actionNew_DB_Window">
   <property name="text">
    <string>New DB Window</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Flow_Window">
   <property name="text">
    <string>New Flow Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
